gradle/libs.versions.toml

```diff
@@ -0,0 +1,29 @@
+[versions]
+agp = "8.0.2"
```

Comment:
Will the lib be compatible with projects using AGP < 8.0?)

Reply:
yes